index.html

```diff
@@ -66,6 +66,199 @@ <h3>I'm a Software Developer</h3>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+        
+       
+      </div>
+     
+    </section>
+    <section class="about-myself">
+      <div class="about-me">
+        <h1 class="about-me-heading">About me</h1>
```

Comment:
`HTML` header tags follow a hierarchy, from `<h1>` to `<h6>`.
- `H1` tags are used to denote the most important text, such as the main theme or title of a content.
- `H2` and `H3` tags are commonly used as subheadings.
- Finally, `H4`, `H5`, and `H6` tags may be used to provide further structure within those subsections.

- [ ] Please use `hx` tags (i.e. `h2`, `h3`, `h4`, `h5` , `h6`) instead of `h1` tag for "About me" heading. 

_Note: keep in mind the hierarchy of these tags e.g.` h3` can't come before` h2 `and `h3 ` can't be placed after` h4` etc._